### PR TITLE
Add devcontainer commands to make

### PR DESCRIPTION
**Pull request by relud**
and additional devcontainer related docs to match https://github.com/mozilla-services/eliot/pull/122

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,9 @@ help:
 .docker-build:
 	make build
 
+.devcontainer-build:
+	make devcontainerbuild
```
Comment (smarnach) [inline Makefile:34]: I don't think we need a recursive make invocation.
```suggestion
.devcontainer-build: devcontainerbuild
```

Comment (smarnach) [reply Makefile:34]: Oh, I see, we have the same thing for `.docker-build`. Maybe there was a reason for the recursive invocation? I can't think of anything, though.

**Review by smarnach — APPROVED** [review]
I've always let VS Code build the dev container. Is there any reason to build it manually?

In any case, this looks good to me.

**Comment by relud** [comment]
> I've always let VS Code build the dev container.

I have to manually build the container for everything to work

**Pull request merged**